packages/main/src/Breadcrumbs.js

```diff
@@ -516,12 +538,43 @@ class Breadcrumbs extends UI5Element {
 	 */
 	get _linksData() {
 		const items = this.getSlottedNodes("items").slice(this._overflowSize);
+		const itemsCount = items.length; // get size before removing of current location
 
 		if (this._endsWithCurrentLocationLabel) {
 			items.pop();
 		}
 
-		return items.filter(item => this._isItemVisible(item));
+		return items
+			.filter(item => this._isItemVisible(item))
+			.map((item, index) => {
+				const { accessibleNameRef, accessibleName } = this._getItemAccessibleNameData(item, index + 1, itemsCount);
+
+				item.accessibleNameRef = accessibleNameRef;
+				item._accessibleNameText = accessibleName;
+
+				return item;
+			});
+	}
+
+	/**
+	 * Getter for accessible name of the current location. Includes the position of the current location and the size of the breadcrumbs
+	 */
+	get _currentLocationAccName() {
+		let items = this.getSlottedNodes("items").slice(this._overflowSize);
+		items = items.filter(item => this._isItemVisible(item));
```

Comment:
i saw that we are using doing this filter more than once. can we add new getter called visibleItems or something like that. this is not related to you change. it's just idea for refactoring